src/RichTextEditor.js

```diff
@@ -45,6 +46,7 @@ type Props = {
   handleReturn?: (event: Object) => boolean;
   readOnly?: boolean;
   disabled?: boolean; // Alias of readOnly
+  toolbarConfig?: EditorToolbarConfig;
```

Comment:
An example would be indeed helpful. I'm not used to flow type annotations. Thanks for the great code review!